package.json

```diff
@@ -7,7 +7,7 @@
     "watch": "npm run watch-other && npm run watch-js",
     "watch-other": "node bin/watch &",
     "watch-js": "webpack --mode=development --progress --profile --colors --watch",
-    "test": "standard && node test/unit",
+    "test": "standard && node ./bin/log-conventions && node test/unit",
```

Comment:
Let's split this into three subcommands. test-standard, test-unit, test-log-convenctions.

Reply:
feel free to continue this... I'm done for today (still awake from yesterday)

Reply:
I will apply some changes, get some rest :) Will very likely be at brombi tuesday/wednesday